File: Swift/dataTypes.swift
var i8: Int8 = 114;
var i16: Int16=542;
var i32: Int32=245425;
var i64: Int64=6895899852;
var i: Int = 3654; // Platform-sized (32-bit for 32-bit platform, 64-bit for 64-bit platform)

var u: UInt = 3654; // Platform-sized (32-bit for 32-bit platform, 64-bit for 64-bit platform)
var u8: UInt8 = 100;
var u16: UInt16 = 500;
var u32: UInt32=985;
var u64: UInt64=88525474;

var f: Float = 2.42; // Float
var d: Double = 2.9652;
var flag: Bool = true;
var char: Character = "A";
var s: String = "Maruf";
var name: String? = nil;

print("8 bit signed integer", i8);
print("16 bit signed integer", i16);
print("32 bit signed integer", i32);
print("64 bit signed integer", i64);
print("Platform sized signed integer", i);

print("8 bit unsigned integer", u8);
print("16 bit unsigned integer", u16);
print("32 bit unsigned integer", u32);
print("64 bit unsigned integer", u64);
print("Platform sized unsigned integer", u);

print("Float",f)
print("Double",d)
print("Boolean",flag)
print("Character",char)
print("String",s)
print("Null",name as Any)

File: rust/dataTypes.rs
// statically typed

fn main() {
    // Signed
    let sByte: i8 = 127; // 8-bit integer
    let sShort: i16 = 142; // 16-bit integer
    let sInt: i32 = 148; // 32-bit integer
    let sLInt: i64 = 18521; // 64-bit integer
    let sDLInt: i128 = 164684; // 128-bit integer

    // Un-signed
    let uByte: u8 = 145; // 8-bit integer
    let uShort: u16 = 2415; // 16-bit integer
    let uInt: u32 = 96523; // 32-bit integer
    let ulInt: u64 = 56146487; // 64-bit integer
    let uDLInt: u128 = 54616351136; // 128-bit integer

    let flt: f32 = 25.1452; // Float
    let dlft: f64 = 52.1482522; // Double Float
    let c: char = 'A'; // char
    let flag: bool = true; // Boolean

    let tupl: (i32, bool) = (421, true);
    let arr: [i32; 5] = [1, 8, 5, 4, 6];
    let strs = String::from("Amazon");

    println!("sByte: {}", sByte);
    println!("sShort: {}", sShort);
    println!("sInt: {}", sInt);
    println!("sLInt: {}", sLInt);
    println!("sDLInt: {}", sDLInt);
    println!("uByte: {}", uByte);
    println!("uShort: {}", uShort);
    println!("uInt: {}", uInt);
    println!("ulInt: {}", ulInt);
    println!("uDLInt: {}", uDLInt);
    println!("flt: {}", flt);
    println!("dlft: {}", dlft);
    println!("char: {}", c);
    println!("flag: {}", flag);
    println!("tupl: {:?}", tupl);
    println!("arr: {:?}", arr);
    println!("strs: {}", strs);
}
